python-backend/app/api/mines/location/models/mine_map_view_location.py

```diff
@@ -38,4 +38,13 @@ def json_for_map(self):
                 'latitude': str(self.latitude),
                 'longitude': str(self.longitude)
             }
-        }
\ No newline at end of file
+        }
+
+    def compact_for_map(self):
```

Comment:
What's this doing?

Reply:
It was used for testing a more compact representation of the data. It's not used right now, but I didn't see harm in leaving it as it's something I'd still like to work on.

Reply:
If it helps, there's a FLASK flag set which removes any whitespaces and tabs and newlines from our JSON responses

Reply:
The whitespace / tab behavior is overridden by flask_restplus's marshaling. We are already setting restplus to remove whitespace when it does so, but it does add newline's which can't be disabled.

Using delimiters would remove the need to define each property name eg "guid", "latitude", "longitude" over and over again, which is what I was testing. The difference is notable!